### PR TITLE
Better markdown

### PR DESCRIPTION
**Pull request by AppleJuiceNerd**
A better version of the markdown file!

### CODE REVIEW

**Comment by bahmanm** [comment]
That's some very good work!  I'll looking int it in the next couple of days (medical condition w/ my eyes) and leave my feedback here if any.

File: README.md [hunk 1]
```diff
@@ -3,94 +3,97 @@
...
+
+1. Read a CSV file that contains product inventory.
+2. Read a CSV file that contains sales invoice lines.
 3. Check if the product referenced on each line has enough inventory.
-   1. If no, do not import the invoice to which the line belongs.
-   2. If yes, import the invoice line and update the inventory of the product accordingly.
-4. It is important to process the invoices in the order they appear in the CSV.
-5. Print a list of invoices listing the problematic line(s) for each invoice.
+	1. If no, do not import the invoice to which the line belongs.
+	2. If yes, import the invoice line and update the inventory of the product accordingly.
```
Comment (bahmanm) [inline README.md:63]: Can you use 2 or 4 spaces instead of TABs here?

File: README.md [hunk 2]
```diff
@@ -3,94 +3,97 @@
...
 
-1. Read a CSV file into memory containing product inventory.
-2. Read two CSV files into memory, one containing sales invoice lines and the other purchase invoice
-   lines. Each invoice has a timestamp which denotes the transaction date and time.
+> [!IMPORTANT]
+> It is important to process any given invoice only once all the other invoices (sales and purchase) with earlier timestamps have been processed.
+
+1. Read a CSV file that contains product inventory.
+2. Read two CSV files, one containing sales invoice lines and the other containing purchase invoice lines.
+	lines. Each invoice has a timestamp which denotes the transaction date and time.
```
Comment (bahmanm) [inline README.md:85]: Ditto re TABs

File: README.md [hunk 3]
```diff
@@ -3,94 +3,97 @@
...
 3. Check if the product referenced on each sales invoice line has enough inventory.
-   1. If no, do not import the sales invoice to which the line belongs.
-   2. If yes, import the invoice line and update the inventory of the product accordingly.
-4. It is important to process any given invoice only once all the other invoices (sales and
-   purchase) with earlier timestamps have been processed.
-5. Print a list of sales invoices listing the problematic line(s) for each invoice.
+	1. If no, do not import the sales invoice to which the line belongs.
+	2. If yes, import the invoice line and update the inventory of the product accordingly.
```
Comment (bahmanm) [inline README.md:88]: ditto RE TABs

**Review by bahmanm — COMMENTED** [review]
Nice Work :clap:  

I only had a minor comment RE TABs but LGTM otherwise :rocket:

File: README.md (hunk 1 above)
Comment (AppleJuiceNerd) [reply README.md:63]: I do kind of prefer tabs, so I changed everything to tabs 😅
I'll change everything back 👍

Comment (AppleJuiceNerd) [reply README.md]: I think that's everything.

**Comment by bahmanm** [comment]
Sweet!  Thanks @AppleJuiceNerd :pray:

**Pull request merged**